lib/web_ui/dev/generate_builder_json.dart

```diff
@@ -154,7 +154,7 @@ Iterable<dynamic> _getTestStepsForPlatform(
         'test_dependencies': <dynamic>[
           <String, dynamic>{
             'dependency': 'goldctl',
-            'version': 'git_revision:dddc0623e63150cbbafdcb273d4048f329e1dd09',
+            'version': 'git_revision:720a542f6fe4f92922c3b8f0fdcc4d2ac6bb83cd',
```

Comment:
@eyebrowsoffire can we get this value from `.ci.yaml` instead of hard-coding it here?

Reply:
Yeah I think it was better when we kept this version in one single source of truth in the recipes. Between here and the ci.yaml in the framework (updated in https://github.com/flutter/flutter/pull/137441), there were more than 7 different version strings. 😕 
Tracking all these down feels like it was better when there one one place this was updated.

Reply:
Answering the specific question, no, there is not a way to specify these from `.ci.yaml` currently. Only the orchestrator CI step is actually specified in the `.ci.yaml` file. This generates the builder JSON file that specifies the `engine_v2` sub-builds, so we have to specify that here. I do understand the distaste for the deep layering of our CI processes, but it's just the way things are at the moment.

Reply:
My recommendation for the engine repo would be to pull the CIPD package with `gclient sync` by listing it in the DEPS file. Then at least for the engine repo, the hash would be in only one place. The tricky bit would be that the scripts that use goldctl would need to be updated to refer to the destination of the CIPD package in the engine tree.

Reply:
I think that sounds good. I am not at all familiar with the engine implementation of Gold, maybe @mdebbar can advise on changing it to use `gclient sync`? I'd like to land this for now since it is trying to resolve a high number of flakes we are seeing from the service.